apps/wallet/src/ui/app/components/SelectDropdown/index.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ChevronDown12 } from '@mysten/icons';
+import * as Select from '@radix-ui/react-select';
+import { Text } from '_app/shared/text';
+
+export interface SelectDropdownProps {
+	placeholder?: string;
+	dropdownOptions?: string[];
```

Comment:
I think `options` is probably sufficient, I don't think the `dropdown` prefix gives us anything here.